image.go

```diff
@@ -20,6 +20,8 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Image provides access to internal image info
```

Comment:
Instead of add a `Image` or `Info` function return the whole internal image object.
I prefer adding `Image.Config`, `Image.Size`, `Image.ChainID`/`Image.RootFS` instead.

/cc @stevvooe @dmcgowan What do you think?